sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b1"
```

Comment:
This should be b2 right?

Reply:
Yes, it should be b2. Weird, it auto update after regenerating. cc: @iscai-msft

Reply:
Reverted.